[Feat] Enable provision of theme-specific logos (#745)
mckinsey/vizro: vizro-core/docs/pages/user-guides/assets.md

## Add a logo image

### Single logo

> Vizro automatically incorporates a logo in the top-left corner if an image named `logo.<extension>` is present in the

Reviewer: ```suggestion
Vizro automatically incorporates a logo in the top-left corner of an image named `logo.<extension>` is present in the
```

Author: Ah, I see that this might have been confusing. I think above is technically not correct though, but would you agree with the following:

Vizro automatically incorporates a logo in the top-left corner **of a dashboard** **if** an image named `logo.<extension>` is present in the ..

Reviewer: Ah, I see. I thought it was unlike you to submit a PR with a typo 🙇 

Let's invert the sentence.

> Vizro automatically incorporates a logo in the top-left corner if an image named `logo.<extension>` is present in the

Reviewer: ```suggestion
If an image named `logo.<extension>` is present in the assets folder, Vizro automatically incorporates the logo in the top-left corner of the dashboard.
```

Author: That's perfect

> assets folder.

Reviewer: ```suggestion

```

### Theme-specific logos

You can also supply two images named `logo_dark.<extension>` and `logo_light.<extension>` to switch logos
based on the theme (dark or light).

> Note that both `logo_dark` and `logo_light` must be provided together, and they cannot be combined with a single `logo`.

Reviewer: ```suggestion
Note that both `logo_dark` and `logo_light` must be provided together, and they cannot be combined within a single `logo`.
```

Reviewer: Would it make sense to show a screen grab to illustrate what the logo looks like?

Author: I like that idea! 👍 

Above is technically incorrect as well, but that's my fault, because I don't think it's clear from my initial draft 😅 
I don't mean that logo_dark and logo_light cannot be combined within a single logo, actually only one will always be visible (depending on which theme is active), so they will be "merged" into one logo.

But the user has the following options:

**Valid Configurations**

- Single Logo: Only logo.<extension> -> will be used for dark and light theme
- Theme Logos: Both logo_dark.<extension> and logo_light.<extension> -> will be switched out for dark / light theme
- No Logo: No logo images provided

And what I meant with the sentence above is that logo_dark.svg and logo_light.svg cannot be provided with logo.svg. How would you phrase that? 😬

Reviewer: I've made a suggestion!

Author: That's so much better 😄 Thanks for making my explanations clear! 🚀
